src/util/stream-to-promise.js

```diff
@@ -0,0 +1,9 @@
+/* @flow */
+
+export async function streamToPromise(
```

Comment:
If we are just using stream-to-promise to wait for file closing, would it be better to switch to something like [event-to-promise](https://www.npmjs.com/package/event-to-promise)? So that we can explicitly specify what we are waiting, i.e. a `close` event.